blaze-core/src/test/scala/org/http4s/blazecore/util/Http1WriterSpec.scala

```diff
@@ -253,13 +257,13 @@ class Http1WriterSpec extends Http4sSpec with CatsEffect {
     }
 
     "write a deflated resource" in {
-      val p = resource.through(deflate())
-      (p.compile.toVector.map(_.toArray), DumpingWriter.dump(resource.through(deflate())))
+      val p = resource.through(deflate(DeflateParams.DEFAULT))
+      (p.compile.toVector.map(_.toArray), DumpingWriter.dump(resource.through(deflate(DeflateParams.DEFAULT))))
         .mapN(_ === _)
     }
 
     "must be stack safe" in {
-      val p = repeatEval(IO.async[Byte](_(Right(0.toByte)))).take(300000)
+      val p = repeatEval(IO.pure[Byte](0.toByte)).take(300000)
```

Comment:
Not sure if this is right.

Reply:
I think that's fine?  It's making 300000 chunks either way.